### PR TITLE
updated api docs with POST request info

### PR DESCRIPTION
**Pull request by jowlee**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -504,3 +530,109 @@ Example Response:
...
+  content: 'Content goes here',
+  title: 'Some Title'
+}
+```
+Example Response:
+```javascript
+
+{
+  images: [ './data/eventImages/000000000000000000000005/000000000000000000001005'],
+  authors: ['000000000000000000000006', '000000000000000000000007', '000000000000000000000004', '000000000000000000000008'],
```
Comment (leej42) [inline API.md]: I put random user ID's in here. Currently, our request does not populate this field...

File: API.md [hunk 2]
```diff
@@ -422,6 +422,32 @@ Example Response:
...
+  isInstructor: false
+}
+```
+
+Example Response:
+```javascript
+
+{
+  $promise: {Promise Object},
+  $resolved: true,
```
Comment (jowlee) [inline API.md:445]: Should I include the actual Promise Object? Or are the promise and resolved not necessary?

Comment (seveibar) [inline API.md:445]: :penguin: i probably wouldn't have included it and it actually should just be removed completely from the API haha

**Comment by seveibar** [comment]
I like it- feel free to fix :penguin: retroactively

**Pull request merged**